examples/Makefile

```diff
@@ -38,6 +40,17 @@ list_people_cpp: list_people.cc protoc_middleman
 	pkg-config --cflags protobuf  # fails if protobuf is not installed
 	c++ list_people.cc addressbook.pb.cc -o list_people_cpp `pkg-config --cflags --libs protobuf`
 
+protoc_middleman_d: addressbook.proto
+	mkdir -p tutorial
+	protoc --d_out=tutorial addressbook.proto
+	@touch protoc_middleman_d
```

Comment:
The examples makefile didn't work for me out of the box - it looks like protoc already places the generated `addressbook.d` file in a subdirectory, thus the makefile created `tutorial/tutorial/addressbook.d`. I had to remove the D-specific middleman target to get it to work:

```diff
diff --git a/examples/Makefile b/examples/Makefile
index e65c5cee..3abf7385 100644
--- a/examples/Makefile
+++ b/examples/Makefile
@@ -16,7 +16,7 @@ clean:
 	rm -f javac_middleman AddPerson*.class ListPeople*.class com/example/tutorial/*.class
 	rm -f protoc_middleman addressbook.pb.cc addressbook.pb.h addressbook_pb2.py com/example/tutorial/AddressBookProtos.java
 	rm -f *.pyc
-	rm -f protoc_middleman_d add_person_d list_people_d
+	rm -f add_person_d list_people_d
 	rm -f protoc_middleman_go tutorial/*.pb.go add_person_go list_people_go
 	rmdir tutorial 2>/dev/null || true
 	rmdir com/example/tutorial 2>/dev/null || true
@@ -24,7 +24,7 @@ clean:
 	rmdir com 2>/dev/null || true
 
 protoc_middleman: addressbook.proto
-	protoc --cpp_out=. --java_out=. --python_out=. addressbook.proto
+	protoc --cpp_out=. --java_out=. --python_out=. --d_out=. addressbook.proto
 	@touch protoc_middleman
 
 protoc_middleman_go: addressbook.proto
@@ -40,15 +40,10 @@ list_people_cpp: list_people.cc protoc_middleman
 	pkg-config --cflags protobuf  # fails if protobuf is not installed
 	c++ list_people.cc addressbook.pb.cc -o list_people_cpp `pkg-config --cflags --libs protobuf`
 
-protoc_middleman_d: addressbook.proto
-	mkdir -p tutorial
-	protoc --d_out=tutorial addressbook.proto
-	@touch protoc_middleman_d
-
-add_person_d: add_person.d protoc_middleman_d
+add_person_d: add_person.d protoc_middleman
 	dmd -gc -I../d/src add_person.d tutorial/addressbook.d ../d/src/google/protobuf/*.d -ofadd_person_d
 
-list_people_d: list_people.d protoc_middleman_d
+list_people_d: list_people.d protoc_middleman
 	dmd -gc -I../d/src list_people.d tutorial/addressbook.d ../d/src/google/protobuf/*.d -oflist_people_d
 
 add_person_go: add_person.go protoc_middleman_go
```

Reply:
corrected. Good catch! I had a strange `PATH` on my system.